src/views/400.njk

```diff
@@ -0,0 +1,15 @@
+{% extends 'layout.njk' %}
+{% block pageTitle %}
+  {{siteTitle}} - Bad request 
+{% endblock %}
+{% block content %}
+  <div class="govuk-width-container">
+    <div class="govuk-main-wrapper--auto-spacing">
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+          <h1 class="govuk-heading-xl">Bad request.</h1>
```

Comment:
Remove period (isn't in the equivalent place on Page not Found).

Reply:
I would leave it for now and fix it next time for this long standing PR if you agree.